CloudFabric.Projections/Queries/SortInfo.cs

```diff
@@ -0,0 +1,35 @@
+namespace CloudFabric.Projections.Queries;
+
+public class SortInfo
```

Comment:
This class basically duplicated ProjectionQuery class in the same folder. Which also has `OrderBy` property. Was there a reason to create new class?